### PR TITLE
feat/pageClassroom

### PR DESCRIPTION
**Pull request by mouracamila**
create the page structure #149 
classroom progress widget #150 
classroom accordion #153

### CODE REVIEW

File: src/modules/Classroom/containers/ClassroomView.tsx [hunk 1]
```diff
@@ -1,14 +1,125 @@
+import { Accordion, Button } from 'flowbite-react';
```
Comment (arianerocha) [inline src/modules/Classroom/containers/ClassroomView.tsx:1]: can you move this into a `Classroom/containers/Classroom/index.ts` or `Classroom/containers/ClassroomOverview/index.ts` to keep the structure consistent with the course module?

File: src/modules/Classroom/containers/ClassroomView.tsx [hunk 2]
```diff
@@ -1,14 +1,125 @@
...
 };
 
 const ClassroomView: React.FC = () => {
   const params = useParams() as RouteParams;
 
-  return <>{JSON.stringify(params)}</>;
+  const cardReminders: CardReminderProps[] = [
+    {
+      title: 'Exames finais',
+      data: '18/02/2022',
```
Comment (arianerocha) [inline src/modules/Classroom/containers/ClassroomView.tsx:22]: Please change it to something like
````
{
  title: '',
  reminderAt: ''
}
````

to make it easy to extract to the mock server later.

File: src/modules/Classroom/components/CardReminder/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { HiChevronRight } from 'react-icons/hi';
+
+export interface CardReminderProps {
+  title: string;
+  data?: string;
```
Comment (arianerocha) [inline src/modules/Classroom/components/CardReminder/index.tsx:6]: maybe `reminderAt` or `reminderDate` ?

**Review by arianerocha — CHANGES_REQUESTED** [review]

**Comment by rluders** [comment]
Drop

**Pull request closed without merging**